docs/source/tutorial/package_a_project.md

```diff
@@ -111,29 +111,19 @@ The resulting `.whl` packages only contain the Python source code of the Kedro p
 
 The project configuration is provided separately in a `tar.gz` file, also inside the `dist` folder. This compressed version of the config files excludes any files inside the `local` directory.
 
-### Package recipients
+### Run a packaged project
 
-Recipients of the `.whl` file need to have Python and `pip` on their machines, but do not need to have Kedro installed.
-
-A recipient can install the project by calling:
+To run a packaged project it must first be installed. To install the project, run the following command:
 
 ```bash
 pip install <path-to-wheel-file>
 ```
 
-An executable, `spaceflights`, is placed in the `bin` subfolder of the Python install folder, so the project can be run as follows:
-
-```bash
-python -m spaceflights
-```
+Once your project is installed, it can be run either using the project's CLI or interactively using Python code.
 
-```{note}
-The recipient will need to add a `conf` subfolder. They also need to add `data` and `logs` if the pipeline loads/saves local data or uses logging.
-Alternatively, they can make use of the ``OmegaConfigLoader`` to run the configuration directly from the compressed .tar.gz configuration file by running
-kedro run --conf-source <path-to-compressed-config>.tar.gz
-```
+To run your project using its CLI, run `<project_name>`. To see a full list of the options that the project's CLI supports, you can run `<project_name> --help`.
```

Comment:
Worth adding an example snippet right below?

Reply:
I thought about this, but it is difficult to justify as the commands are so short. Willing to be overruled but I would keep it as is.